integrations/cassandra/src/inventory.go

```diff
@@ -15,16 +15,13 @@ func getInventory() (map[string]interface{}, error) {
 		return nil, err
 	}
 
-	inventory := make(map[string]interface{})
-	err = yaml.Unmarshal(rawYamlFile, &inventory)
-	if err != nil {
-		return nil, err
-	}
+	i := make(map[string]interface{})
```

Comment:
Why we don't use inventory struct: `inventory.Inventory.Item`?

Reply:
However I'm not sure if we're allowed to do it here. It seems that the field is private. If so, then is it really ok to have `item` field not public.